src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java

```diff
@@ -57,8 +60,10 @@ public void ingestPaymentOrders(String externalUserId) {
 
             if (PAYMENT_TYPE_SEPA_CREDIT_TRANSFER.equals(paymentType)) {
                 randomArrangement = getRandomFromList(sepaCtArrangements);
-            } else {
+            } else if (PAYMENT_TYPE_US_DOMESTIC_WIRE.equals(paymentType)) {
```

Comment:
add `|| PAYMENT_TYPE_US_FOREIGN_WIRE` to the condition

Reply:
done :)